### PR TITLE
#16 - Velocity boundary documentation

### PR DESCRIPTION
**Pull request by ezrayst**
- [x] Documentation on `nodal_euler_angles` input format
- [x] Explanation what would happen to the nodes with Euler angles

### CODE REVIEW

File: user/preprocess/ascii-mesh-particles.md [hunk 1]
```diff
@@ -64,11 +64,11 @@ An example of a 3D mesh comprising of two cells and twelve nodes is shown below:
 Velocity constraints at each node can be specified in the following format:
 
 ```
-n_0     d_1     v_1
-n_1     d_0     v_0
+n_0     d_0     v_0
```
Comment (kks32) [inline user/preprocess/ascii-mesh-particles.md]: Is this still valid?

**Review by kks32 — CHANGES_REQUESTED** [review]

File: user/preprocess/ascii-mesh-particles.md (hunk 1 above)
Comment (ezrayst) [reply user/preprocess/ascii-mesh-particles.md]: Now that we will have this option, I think it is still valid.

**Comment by kks32** [comment]
I don't see any changes here!

**Comment by ezrayst** [comment]
I made a mistake in merging. Now we have some changes here!

File: user/preprocess/input.md [hunk 2]
```diff
@@ -395,6 +395,29 @@ Velocity and friction constraints can be specified on the nodes. Velocity constr
...
+n_i     alpha_i     beta_i    gamma_i n_i     alpha_i     beta_i    gamma_i
+... ...
+... ...
+n_n     alpha_n     beta_n    gamma_n n_n     alpha_n     beta_n    gamma_n
+``` ```
+where,  where, 
+`n_i` is the node number. `n_i` is the node number.
+
+
+`alpha_i`, `beta_i` and `gamma_i` are the three Euler angles defined on [this page](https://mpm.cb-geo.com/#/theory/geometry/rotation-matrices).  `alpha_i`, `beta_i` and `gamma_i` are the three Euler angles defined on [this page](https://mpm.cb-geo.com/#/theory/geometry/rotation-matrices).
```
Comment (kks32) [inline user/preprocess/input.md]: This line is repeating.

**Review by kks32 — CHANGES_REQUESTED** [review]
What about 2D?

File: user/preprocess/input.md (hunk 2 above)
Comment (ezrayst) [reply user/preprocess/input.md]: Fixed!

**Comment by ezrayst** [comment]
2D, let `gamma` be zero. I have added that in the documentation as well.

**Review by kks32 — APPROVED** [review]

**Pull request merged**